File: PHP/1-Foundation/8-Palindromes/code.php
<?php

function is_palindrome($word) {
   for ($i = 0; $i < strlen($word) / 2; $i++) {
      if ($word[$i] !== $word[strlen($word) - $i - 1]) {
         return false;
      }
   }
   return true;
}


echo 'Enter a word: ';
$word = rtrim(fgets(STDIN));

if (is_palindrome($word)) {
   echo 'Yes';
}
else {
   echo 'No';
}

File: Python/01-Foundation/7-Palindromes/code.py
def is_palindrome(s):
   n = len(s)
   for i in range(n // 2):
      if s[i] != s[n - i - 1]:
         return False
   return True


s = input('Enter a word: ')

if is_palindrome(s):
   print('Yes')
else:
   print('No')